extras/cronwrapper

```diff
@@ -14,11 +14,11 @@
 # LOGGING if true, logs all output to syslog daemon facility
 #         (typically /var/log/daemon.log or /var/log/syslog)
 #
-# Set CONFIGURED to true once you've setup the previous three
-# options.
+# NOTIFYONRETURN is used to define which return codes from plexupdate
```

Comment:
This isn't entirely accurate. It's which **one** return code will notify you. But I think this might be overthinking it a bit honestly. :) As I said, this use case seems like a pretty unlikely one anyway, so I'd suggest just leaving it as `if [ $RET -ne 0 ]` until we hear from someone who actually doesn't want that.